### PR TITLE
Changed the classes on this thing to match the styles from steps.

### LINKED ISSUE #6913: Changing the language in the upload photo section
#### User Story

As a member, I'd like to have more context about what the photo uploader is and that someone is reviewing my photo. As an admin, I want the member to know that the photo will be reviewed and that the photo is the scholarship entry.
#### Additional Information

For campaigns that have scholarships on them:
—— scholarship, no RB: “Add your photo here for a scholarship entry!”
—— scholarship, RB: “Add another photo and we’ll check it out!”

For campaigns that don't have a scholarship on them:
— — no scholarship, no RB “Add your photo here and we’ll check it out!”
—— no scholarship, RB “Add another photo and we’ll check it out!"
#### Why This Matters

We want to add clarity around the RB process -- a part of that is clarifying why the photo is important and that someone actually looks at them!
#### Examples

screen shot 2016-08-22 at 10 27 08 am

screen shot 2016-08-22 at 10 28 22 am

### LINKED ISSUE #6930: "Join Others" title styling is incorrect
The styling of the new title in Step 4 is incorrect. It's currently styled as an all caps version of the interior campaign title, but it should have the same weight as a Step X header. It should be slightly larger and bolder.

Current:
screen shot 2016-08-23 at 11 46 19 am

Desired:
example

### PR DESCRIPTION
**Pull request by angaither**
#### What's this PR do?

bow's down to @lkpttn's #fender demands.
#### How should this be reviewed?

🎱 
#### Any background context you want to provide?

I can look at a mock up and everything will look exactly the same to me. 
#### Relevant tickets

Fixes #6930

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php [hunk 1]
```diff
@@ -343,7 +343,7 @@
               <p class="copy inline-alt-text-color"><?php print $reportback_copy; ?></p>
             <?php endif; ?>
             <?php //@TODO @fenders look into adding a .-yelling class that all caps things ?>
-            <h3 class="inline-alt-text-color"><?php print t('JOIN OTHERS MAKING A DIFFERENCE'); ?></h3>
+            <h2 class="heading -emphasized"><?php print t('JOIN OTHERS MAKING A DIFFERENCE'); ?></h2>
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php]: The [`.heading.-emphasized`](http://forge.dosomething.org/#heading) style modifier makes things all-caps as part of its visual appearance, so you can remove the `@TODO` on the line above and make the text sentence case again. ✏️

Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php]: One Q (perhaps for @lkpttn): do we want the emphasized heading's bottom-border to be visible here on the yellow Step 4 background? If so, we should add the `.-inverse` modifier to make it white.

Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php]: If not, it feels like this is perhaps a whole new style variant? Nothing we need to necessarily address in this PR, but worth thinking about if we see this all-caps no-visible-underline heading style appear elsewhere.

Comment (angaither) [reply lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php]: I will leave that up to you and sir @lkpttn 🎨

**Pull request merged**